src/bp/ui-studio/package.json

```diff
@@ -67,6 +67,7 @@
     "react-jsonschema-form": "^1.8.1",
     "react-loaders": "^3.0.1",
     "react-markdown": "^4.3.1",
+    "react-monaco-editor": "^0.36.0",
```

Comment:
The editor is heavy. I think it would be a better idea to code it in the module code-editor, and export it as a LiteEditor, like we do on the NLU module.

Reply:
We'll use the code editor in studio in triggers and execute nodes ... if you disable code editor module, those still need to work

Reply:
tbh I'm not sure if we'll be able to use a code editor in the nodes, I tried to implement, but the monaco editor isn't meant to take the size of the content + grow as you type and making it do it result in incredibly slow performance and poor user experience. We can look more into it later, for now it will be a textearea, just forgot to remove this from the package

Reply:
Let's display the rich editor if you have the code-editor module, and fallback on a simple textarea otherwise. The code editor is the heaviest module at 25mb, let's not add that to the studio (or we move it to the core)